python/orca/src/bigdl/orca/learn/pytorch/callbacks/tqdm.py

```diff
@@ -53,6 +53,29 @@ def before_train_epoch(self, runner):
                                         unit="batch",
                                         leave=False)
 
+    def after_val_iter(self, runner):
+        if self._is_rank_zero(runner):
+            runner._progress_bar.n = runner.batch_idx
+            postfix = {}
+            if "val_loss" in runner.validation_stats:
+                postfix.update(loss=runner.validation_stats["val_loss"])
+            runner._progress_bar.set_postfix(postfix)
+
+    def before_val_epoch(self, runner):
+        if self._is_rank_zero(runner):
+            desc = "1/1e"
+
+            invalidInputError(tqdm is not None,
+                              "tqdm is not installed, please install with 'pip install tqdm'")
+            runner._progress_bar = tqdm(total=len(runner.val_loader),
+                                        desc=desc,
+                                        unit="batch",
+                                        leave=False)
+
+    def after_pred_iter(self, runner):
+        if self._is_rank_zero(runner):
+            print("\r rank0 predict batch_idx: {%d}" % runner.batch_idx, end="")
```

Comment:
remove `rank0`? Just Predict batch_idx: ...?
For users they don't even know rank0

Reply:
sure